app/models/forms/benefits_evidence.rb

```diff
@@ -7,9 +7,14 @@ def self.permitted_attributes
     define_attributes
 
     validates :correct, inclusion: { in: [true, false] }
+    validate :isnt_blank
```

Comment:
Looks fine, but wouldn't 
  validates :correct, presence: true 
have done the same thing?